### ISSUE TITLE
false positive: MDCPutWithoutRemove

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#653]
```java
import org.slf4j.MDC;

class Foo {

    public String handle(String detail, String code) {
        try {
            setMDC(code, detail);
            return doSomething();
        } finally {
            clearMDC();
        }
    }

    private String doSomething() {
        return "OK";
    }

    private void setMDC(String code, String detail) {
        MDC.put("code", code); // false positive
        MDC.put("detail", detail); // false positive
    }

    private void clearMDC() {
        MDC.remove("code");
        MDC.remove("detail");
    }
}
```
